src/modules/Dropdown/Dropdown.js

```diff
@@ -747,7 +802,13 @@ export default class Dropdown extends Component {
         active={isActive(opt.value)}
         onClick={this.handleItemClick}
         selected={selectedIndex === i}
-        text={opt.text}
+        text={
+          allowAdditions
+          && (additionPosition === 'top' ? i === 0 : i === options.length - 1)
+          && opt.text.startsWith('Add: ') ?
+          <span><strong><em>Add:</em></strong> {opt.text.substring(5)}</span> :
+          opt.text
+        }
```

Comment:
Agreed that it adds a lot of bulk for a simple styling thing. Maybe instead of the `Add: newValue`, we just want to show `newValue` so that it looks like one of the other options. Or again make it optional (introduces another prop though).

I think without other styling it's a little confusing that your custom value is prepended by `Add:`. Might be just me though.